src/widgets/PlotWidget.zig

```diff
@@ -577,6 +587,56 @@ pub fn line(self: *PlotWidget) Line {
     };
 }
 
+pub const BarOptions = struct {
+    x: f64,
+    y: f64,
+    w: f64,
+    h: f64,
+    color: dvui.Color,
```

Comment:
Can color be an optional and if it's not given we use dvui.themeGet().focus?

Reply:
Went ahead and made the color optional and made it use `dvui.themeGet().focus` by default.